data/governments.txt

```diff
@@ -1260,7 +1260,9 @@ government "Kraz Cybernetics"
 	swizzle 2
 	color "governments: Free"
 	"player reputation" 0
-	"friendly hail" "friendly free worlds"
```

Comment:
Is this intentionally removing the friendly hails?

Reply:
Probably should have discussed this a bit more in the opening post.
Does it make sense for the Kraz ship to hail like a FW ship? "We are the protectors of this system." but "we" doesn't work in the first place - and this ship hardly even has weapons.
But I don't think the merchant hails exactly work either... Barmy Edward is probably an employer who doesn't let his employees go three weeks away.
It's a complicated question, and I'd be interested in hearing your thoughts on it.

Reply:
Those are good points. There really should be some more corporate hails. Things along the lines of the regular friendly merchant ones, but adjusted to reflect coming from an employee in a medium company. 

I think in this case, probably sticking with the friendly merchant ones is better than not having any, but having a culled subset of those that fits better would be good.

Reply:
Alright. Added a `friendly kraz cybernetics` hail that should resolve this.